modules/ROOT/pages/authentication-authorization/sso-integration.adoc

```diff
@@ -253,6 +253,112 @@ dbms.security.oidc.mysso.claims.groups=roles
+
+When the xref:configuration/configuration-settings.adoc#config_dbms.security.require_local_user[`dbms.security.require_local_user`] configuration setting is set to `true`, users can *only* authenticate when there is a user in the database with an Auth Provider which links to the provider that the user is trying to authenticate with.
+
+If there is no matching Auth Provider, the user will not be able to authenticate or authorize. This applies to all providers.
+
+Conversely, when xref:configuration/configuration-settings.adoc#config_dbms.security.require_local_user[`dbms.security.require_local_user`] is set to `false`, users' Auth Providers will have no bearing on the way that they are authenticated and authorized, instead authentication and authorization will be controlled centrally (for all users) by the database configuration.
+======
+
+
+.Create a user with two Auth Providers allowing the user to authenticate and authorize with either of two providers.
+======
+----
+CREATE USER jake
+SET HOME DATABASE anotherDb
+SET AUTH 'oidc-mysso1' {SET ID 'jakesUniqueMySso1Id'} // `jakesUniqueMySso1Id` must match the value of the claim that you configured via dbms.security.oidc.mysso1.claims.username
+SET AUTH 'oidc-mysso2' {SET ID 'jakesUniqueMySso2Id'} // `jakesUniqueMySso2Id` must match the value of the claim that you configured via dbms.security.oidc.mysso2.claims.username
+----
+This will create a user who can authenticate and authorize using `mysso1` or `mysso2`.
+This example also illustrates that the user can have their home database set even when using only external auth providers.
+======
+
+
+.Alter a user to remove one of their Auth Providers.
+======
+----
+ALTER USER jake
+REMOVE AUTH 'oidc-mysso2'
+----
+This will prevent the user from being able to authenticate and authorize with the `mysso2` provider.
+======
+
+.Alter a user to allow them to authenticate and authorize using username and password
+======
+----
+ALTER USER jake
+SET AUTH 'native' {SET PASSWORD 'changeme' SET PASSWORD CHANGE REQUIRED}
+----
+This will allow the user to authenticate and authorize using the specified username and password (in addition to what they are already configured to use).
+======
+
+
+.Configure the database to allow authentication via `mysso` and authorization via the `native` provider.
+======
+
+Set the following database config:
+[source,configuration,role="noheader"]
+----
+dbms.security.authentication_providers=oidc-mysso
+dbms.security.authorization_providers=native
+----
+
+Then create a user with a `mysso` Auth Provider:
+----
+CREATE USER jake
+SET AUTH 'oidc-mysso' {SET ID 'jakesUniqueMySsoId'} // `jakesUniqueMySsoId` must match the value of the claim that you configured via dbms.security.oidc.mysso.claims.username
+----
+
+Then natively grant the `READER` role to the user:
+----
+GRANT ROLE READER TO alice
```

Comment:
This doesn't make sense, as the examples on this SSO page are using the user jake, chaning alice's roles doesn't follow the example that is being provided.   I think this was a copy/paste from the ldap examples that use the user alice.   We should change this to say. "GRANT ROLE READER TO jake"

Reply:
thanks @b33f. Done.